examples/cluster/plot_adjusted_for_chance_measures.py

```diff
@@ -3,44 +3,45 @@
 Adjustment for chance in clustering performance evaluation
 ==========================================================
 
-The following plots demonstrate the impact of the number of clusters and
-number of samples on various clustering performance evaluation metrics.
+The following plots demonstrate the impact of the number of clusters and number
+of samples on various clustering performance evaluation metrics.
 
-Non-adjusted measures such as the V-Measure show a dependency between
-the number of clusters and the number of samples: the mean V-Measure
-of random labeling increases significantly as the number of clusters is
-closer to the total number of samples used to compute the measure.
+Non-adjusted measures such as the V-Measure show a dependency between the number
+of clusters and the number of samples: the mean V-Measure of random labeling
+increases significantly as the number of clusters is closer to the total number
+of samples used to compute the measure.
 
-Adjusted for chance measure such as ARI display some random variations
-centered around a mean score of 0.0 for any number of samples and
-clusters.
+Adjusted for chance measure such as ARI display some random variations centered
+around a mean score of 0.0 for any number of samples and clusters.
 
-Only adjusted measures can hence safely be used as a consensus index
-to evaluate the average stability of clustering algorithms for a given
-value of k on various overlapping sub-samples of the dataset.
+Only adjusted measures can hence safely be used as a consensus index to evaluate
+the average stability of clustering algorithms for a given value of k on various
+overlapping sub-samples of the dataset.
 
 """
 
 # Author: Olivier Grisel <olivier.grisel@ensta.org>
 # License: BSD 3 clause
 
+# %%
+# We first define a function that computes the score for 2 random uniform
+# cluster labelings.
+#
+# Both random labelings have the same number of clusters for each value possible
+# value in `n_clusters_range`. When fixed_n_classes is not None the first
+# labeling is considered a ground truth class assignment with fixed number of
+# classes.
+#
+# A second function computes de `Adjusted mutual information
+# <https://en.wikipedia.org/wiki/Adjusted_mutual_information>`_.
+
 import numpy as np
-import matplotlib.pyplot as plt
-from time import time
 from sklearn import metrics
 
 
 def uniform_labelings_scores(
     score_func, n_samples, n_clusters_range, fixed_n_classes=None, n_runs=5, seed=42
 ):
```

Comment:
Instead of defining a complex function that generate results for both experiment, we could define 2 simple functions, one in each experiment, just before it is used.

This would make the example easier to follow.